wfe/wfe.go

```diff
@@ -2565,3 +2584,125 @@ func (wfe *WebFrontEndImpl) processRevocation(
 	})
 	return nil
 }
+
+// Verify the External Account Binding in the request and return the same JSON
+// object that was given in the request if successful. If no External Account
+// Binding was given then return nil however if it is required by the server
+// then error.
+func (wfe *WebFrontEndImpl) verifyEAB(newAcctReq newAccountRequest,
+	outerPostData *authenticatedPOST) (*acme.JSONSigned, *acme.ProblemDetails) {
```

Comment:
small nit: When breaking up the function declaration across multiple lines I think it makes sense for each parameter to be on a newline.

e.g.:

```
func (wfe *WebFrontEndImpl) verifyEAB(
        newAcctReq newAccountRequest,
	outerPostData *authenticatedPOST) (*acme.JSONSigned, *acme.ProblemDetails) {
```